Tests/Configuration/ConfigurationBuilderTest.php

```diff
@@ -34,29 +31,29 @@ public function setUp()
     {
         parent::setUp();
 
-        $this->container = new Container();
-        $this->container->addScope(new Scope('request'));
-        $this->container->enterScope('request');
+        $this->container = $this->getMockBuilder('Symfony\Component\DependencyInjection\Container')
+        ->disableOriginalConstructor()
+        ->getMock();
 
         $translator = $this
-            ->getMock('Symfony\Component\Translation\TranslatorInterface');
+            ->createMock('Symfony\Component\Translation\TranslatorInterface');
         $translator
             ->expects($this->any())
             ->method('getLocale')
             ->will($this->returnValue('fr_FR'));
 
-        $this->container->set('translator', $translator);
+        $this->container->setParameter('translator', $translator);
```

Comment:
The translator is a service and not a parameter, so this change should be reverted